docs/technical-reference.md

```diff
@@ -321,7 +321,7 @@ for further details.
     Do not allow wildcard `*` in the login URI redirect. This will be something like the following:
 
         ```
-        https://quilt.<MY_COMPANY>.com/
+        https://quilt.<MY_COMPANY>.com/oauth-callback
```

Comment:
Let's get down to one sentence instead of saying the same thing twice. In other words we have `YOUR_QUILT_CATALOG_URL/oauth-callback` right above this, and then again in this section, but with different wording. So let's consolidate into a single, clear statement that is consistent with the rest of the docs. We probably don't need a code block for this.

Another issue is the "Do not allow" statement is confusing because that's an actual checkbox. We should call it out as such or omit that statement altogether.
<img width="588" alt="Screenshot 2023-05-16 at 12 02 53 PM" src="https://github.com/quiltdata/quilt/assets/1322715/5d881641-72e1-40a6-b6f0-c7dcc02f7819">

Reply:
Clarified the above comment.